admin/tests/unit/controllers/display-languages.spec.js

```diff
@@ -1,4 +1,5 @@
 describe('Display Languages controller', function() {
+  const { DOC_TYPES } = require('@medic/constants');
```

Comment:
I moved this `require` statement inside of the describe block because when it was outside the describe block, I got a conflict between this declaration and the one in `admin/tests/unit/controllers/display-translations.spec.js`.

The clash looked like this:

```
  Uncaught SyntaxError: Identifier 'DOC_TYPES' has already been declared
  at tests/unit/controllers/display-translations.spec.js:1:1

  SyntaxError: Identifier 'DOC_TYPES' has already been declared
```

Reply:
Thanks for the explanation. 

I have a slight preference for the destructuring in the import statement because I feel it makes the access of the constants in the file just a little more concise.

However, if you feel that something like `CONSTANTS.DOC_TYPES.TRANSLATIONS` is easier to read or reason about than `DOC_TYPES.TRANSLATIONS`, I'm happy to change it.